server/commands/admin.py

```diff
@@ -34,9 +34,14 @@ def ooc_cmd_motd(client, arg):
     Show the message of the day.
     Usage: /motd
     """
-    if len(arg) != 0:
-        raise ArgumentError("This command doesn't take any arguments")
-    client.send_motd()
+    if arg == '':
+        client.send_motd()
+    elif client.is_mod:
+        client.server.config['motd'] = arg
```

Comment:
how to make newlines
also wouldn't it be better to just fix /refresh

Reply:
refresh does load the MOTD correctly but maybe i don't want to change the config file every time